src/vm/arm64/PInvokeStubs.asm

```diff
@@ -137,16 +136,6 @@ __PInvokeGenStubFuncName SETS "$__PInvokeGenStubFuncName":CC:"_RetBuffArg"
 ;
         PINVOKE_STUB GenericPInvokeCalli, x15, x12, {true}
 
```

Comment:
There's logic related to this in the actual stub generation macro (above in this file) that should be cleaned up too. Look for where it checks whether the $VASigCookieReg is x1.